src/dbFirebase.js

```diff
@@ -83,15 +82,18 @@ async function fetchPhotos() {
 
 function saveMetadata(data) {
   data.location = new firebase.firestore.GeoPoint(data.latitude, data.longitude);
-  data[config.PHOTO_FIELD.name]= config.PHOTO_FIELD.type === enums.TYPES.number ? Number(data.field) : data.field
+  delete data.latitude;
```

Comment:
why ? (there is a `_.pick` in https://github.com/Geovation/photos/pull/352/files#diff-dceed32b2ff5017478ca834e742de743R97 choosing what to save)

Reply:
I just tried to make the code a bit more clear. The _.pick is still in the code.